gulpfile.js

```diff
@@ -32,7 +33,7 @@ gulp.task('nodemon', function () {
 		script: 'server.js',
 		nodeArgs: ['--debug'],
 		ext: 'js,html',
-		watch: _.union(defaultAssets.server.views, defaultAssets.server.allJS, defaultAssets.server.config)
+		watch: _.union(defaultAssets.server.gulpConfig, defaultAssets.server.views, defaultAssets.server.allJS, defaultAssets.server.config)
```

Comment:
Does Gulp work with new code if you have it watching? I don't remember it being able to do that..